addons/nuxeo-retention/nuxeo-retention-core/src/main/resources/OSGI-INF/retention-pageproviders.xml

```diff
@@ -100,8 +101,8 @@
         <quickFilter name="hasNotLegalHold">
           <clause>ecm:hasLegalHold = 0</clause>
         </quickFilter>
-        <quickFilter name="exceptTrashed">
-          <clause>ecm:isTrashed = 0</clause>
+        <quickFilter name="includeTrashed">
```

Comment:
Isn't it breaking the backward compatibility? It changes the quick filters returned by the REST API

Reply:
For the moment, we have only one client using the addon, and they use the default UI. I can let the original quickfilter and hardcode the expected filters (and removing the exceptTrashed one) in the nuxeo-result element (https://github.com/nuxeo/nuxeo-web-ui/blob/master/elements/nuxeo-results/nuxeo-results.js#L275). What would be the best option?

Reply:
Guess it could be an option, indeed.

But, if we are sure about the impact (only one client and being a "private" addon), would it be that bad to perform the change? (cc @troger )
Of course,  stating that on the "Upgrade Notes" and double checking our impact assessment.

Reply:
If you know what you are doing and the impacts, I'm OK, just wanted to mention it :)

Reply:
TBH, I don't know 🤷‍♂️  😅 
But @gcarlin's input may be a "game changer". Anyway, the other option (safer one, keeping the existing quickFilter) is also viable.

Reply:
Using "hardcoded" quickfilters on the nuxeo-results element also implies that a new quickfilter won't be displayed automatically. That said, I would opt for the safe option (never know if anyone else is using it).

Reply:
But in that case we would not be "hardcoding" since it is a parameter and the users can change it if they want! Or am I seeing it a wrong way?

Reply:
Yes yes, but I guess that's the only part in Web UI we explicitly indicate which quickfilters to display (so you might not guess you need also to update the result element)

Reply:
So, according to nelson note (https://github.com/nuxeo/nuxeo/pull/4214#discussion_r455803626), we will only add an additional quickfilter